templates/lock.mdx

```diff
@@ -23,12 +23,19 @@ Success: True
 
 ## Solving
 
+<details>
+<summary>Spoilers for lock's solutions</summary>
 The arguments required for solving
```

Comment:
Put a newline here to ensure a paragraph break exists here. Otherwise the paragraph may include the `summary` element and that breaks its semantics.
```suggestion

The arguments required for solving
```